qpid-jms-client/src/main/java/org/apache/qpid/jms/transports/netty/NettyTcpTransport.java

```diff
@@ -214,8 +209,8 @@ public void operationComplete(ChannelFuture future) throws Exception {
                 }
             });
         }
-
-        return group;
+        // returning the channel's event loop: groupRef::ref is multi-threaded
```

Comment:
Just call it the 'event loop groop', its both simpler by being immediately understandable, and less likely to go stale as the field+method names can and will change (e.g as suggested).